Modules/Filtering/LabelMap/test/itkLabelImageToLabelMapFilterTest.cxx

```diff
@@ -131,5 +145,8 @@ itkLabelImageToLabelMapFilterTest(int argc, char * argv[])
 
   conversion->Print(std::cout);
 
+  zeroSizeCase();
```

Comment:
Return value of `zeroSizeCase` is ignored. Maybe make it a void function or do a `return zeroSizeCase();` here?

Reply:
done